platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/constructable/constructors/MerkleDbDataSourceBuilderConstructor.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.merkledb.constructable.constructors;
+
+import com.swirlds.config.api.Configuration;
+import com.swirlds.merkledb.MerkleDbDataSourceBuilder;
+
+@FunctionalInterface
+public interface MerkleDbDataSourceBuilderConstructor {
```

Comment:
Why is it needed? Isn't `BootstrapUtils.setupConstructableRegistryWithConfiguration()` enough?

Reply:
`BootstrapUtils.setupConstructableRegistryWithConfiguration()` and `*Constructor` interfaces are working together -- without those interfaces `ConstructableRegistry` would use a default constructor.